### ISSUE TITLE
Cleanup Redis memory when deconstructing Storage object?

### ISSUE THREAD

**Issue by joker-xii** [ekzhu/datasketch#153]
Hello.
I have noticed that the program needs users to manually cleanup Redis memory. And there exist similar issues about Redis key expiration. 
I wonder if it is ok to add an argument in Redis storage config that explicitly deallocate Redis memory after the object's lifecycle, such as a __del__ function to scan the database and unlink all keys?

Thanks.

**Comment by ekzhu** [comment]
Yes. I think this could be useful. Do you think it would be useful to have a 'clear()' method to explicitly remove all data related to the index? Do you want to start a pull-request on this?

**Comment by joker-xii** [comment]
Thanks for the reply, I will try to raise a PR on this issue
